### PR TITLE
docs(migration): add enhancements and improvements

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: packages/demo/src/app/home/home.component.html [hunk 1]
```diff
@@ -151,6 +198,82 @@ <h5 class="h6 bold">Migrations list</h5>
...
+                      <input
+                        id="generalnaming_cwflicense"
+                        class="form-check-input"
+                        type="checkbox"
+                        [(ngModel)]="migrationAccordionGroupedCheckboxes.general.naming_cwflicense"
+                        (change)="migrationAccordionGroupedCheckboxesChange()"
+                      />
+                      <!-- prettier-ignore -->
+                      <label class="form-check-label" for="generalnaming_cwflicense">
+                          <span class="change-badge bg-danger">breaking</span> Removed <span class="highlight">src/lic/cwf-licene.scss</span> file.
```
Comment (gfellerph) [inline packages/demo/src/app/home/home.component.html:238]:
```suggestion
                          <span class="change-badge bg-danger">breaking</span> Removed <span class="highlight">src/lic/cwf-license.scss</span> file.
```

File: packages/demo/src/app/home/home.component.html [hunk 2]
```diff
@@ -151,6 +198,82 @@ <h5 class="h6 bold">Migrations list</h5>
...
+                        type="checkbox"
+                        [(ngModel)]="migrationAccordionGroupedCheckboxes.general.naming_cwfname"
+                        (change)="migrationAccordionGroupedCheckboxesChange()"
+                      />
+                      <!-- prettier-ignore -->
+                      <label class="form-check-label" for="generalnaming_cwfname">
+                          <span class="change-badge bg-danger">breaking</span> Renamed <span class="highlight">src/cwf.scss</span> file to <span class="highlight">src/core.scss</span>.<br>
+                          <span>We recommend to import the <span class="highlight">core.scss</span> file as follows: <code>@use '@swisspost/design-system-styles/core.scss' as post;</code></span><br>
+                          <span>However, do not forget to switch the namespace in your files from <span class="highlight">cwf</span> to either <span class="highlight">core</span> or <span class="highlight">post</span>!</span>
+                      </label>
```
Comment (gfellerph) [inline packages/demo/src/app/home/home.component.html:256]:
```suggestion
                      </label>
                      <p class="info">Read more on namespaces at the <a href="https://sass-lang.com/blog/the-module-system-is-launched">sass module system intro</a>.</p>
```

File: packages/demo/src/app/bootstrap/components/accordion/accordion-demo-page/accordion-demo-page.component.html [hunk 3]
```diff
@@ -1,24 +1,41 @@
...
   <div class="alert alert-info my-large">
-    <p>For the JavaScript accordion to work properly, you need to import collapse.js from Bootstrap into your component.</p>
+    <p>
+      For the JavaScript accordion to work properly, you need to import collapse.js from Bootstrap
+      into your component.
+    </p>
+  </div>
+  <div class="alert alert-info my-large">
+    <p>
+      The detail summary styles have been removed due to some issues.
```
Comment (gfellerph) [inline packages/demo/src/app/bootstrap/components/accordion/accordion-demo-page/accordion-demo-page.component.html:24]:
```suggestion
      The detail summary styles have been removed due to <a href="https://adrianroselli.com/2019/04/details-summary-are-not-insert-control-here.html">accessibility issues with the &lt;summary&gt; element</a>.
```

**Review by gfellerph — CHANGES_REQUESTED** [review]

File: packages/demo/src/app/home/home.component.html [hunk 4]
```diff
@@ -84,41 +98,80 @@ <h4 class="h5 bold">
...
-        ></code>
+        <ol>
+          <li>Commit all the changes you made untill now!</li>
+          <li>Make sure you are running on a node version >= 16!</li>
+          <li>
+            Install the migration package:
+            <code
+              class="d-block mt-1 p-3"
+              [languages]="['bash']"
+              highlight="npm install @swisspost/design-system-migrations"
```
Comment (gfellerph) [inline packages/demo/src/app/home/home.component.html:135]:
```suggestion
              highlight="npm install --save-dev @swisspost/design-system-migrations"
```

**Review by gfellerph — APPROVED** [review]

**Pull request merged**